doc/api/prev_api_changes/api_changes_1.4.x.rst

```diff
@@ -79,112 +80,115 @@ original location:
   :class:`matplotlib.patheffects.PathEffectRenderer` and is now a full
   RendererBase subclass.
 
-* The artist used to draw the outline of a `colorbar` has been changed
-  from a `matplotlib.lines.Line2D` to `matplotlib.patches.Polygon`,
-  thus `colorbar.ColorbarBase.outline` is now a
-  `matplotlib.patches.Polygon` object.
+* The artist used to draw the outline of a `.Figure.colorbar` has been changed
+  from a `matplotlib.lines.Line2D` to `matplotlib.patches.Polygon`, thus
+  `.colorbar.ColorbarBase.outline` is now a `matplotlib.patches.Polygon`
+  object.
 
 * The legend handler interface has changed from a callable, to any object
   which implements the ``legend_artists`` method (a deprecation phase will
   see this interface be maintained for v1.4). See
   :doc:`/tutorials/intermediate/legend_guide` for further details. Further legend changes
   include:
 
-   * :func:`matplotlib.axes.Axes._get_legend_handles` now returns a generator
-     of handles, rather than a list.
+   * ``matplotlib.axes.Axes._get_legend_handles`` now returns a generator of
+     handles, rather than a list.
 
-   * The :func:`~matplotlib.pyplot.legend` function's "loc" positional
-     argument has been deprecated. Use the "loc" keyword instead.
+   * The :func:`~matplotlib.pyplot.legend` function's *loc* positional
+     argument has been deprecated. Use the *loc* keyword argument instead.
 
-* The rcParams `savefig.transparent` has been added to control
+* The :rc:`savefig.transparent` has been added to control
   default transparency when saving figures.
 
-* Slightly refactored the `Annotation` family.  The text location in
-  `Annotation` is now handled entirely handled by the underlying `Text`
-  object so `set_position` works as expected.  The attributes `xytext` and
-  `textcoords` have been deprecated in favor of `xyann` and `anncoords` so
-  that `Annotation` and `AnnotaionBbox` can share a common sensibly named
+* Slightly refactored the `.Annotation` family.  The text location in
+  `.Annotation` is now entirely handled by the underlying `.Text`
+  object so ``.set_position`` works as expected.  The attributes *xytext* and
+  *textcoords* have been deprecated in favor of *xyann* and *anncoords* so
+  that `.Annotation` and `.AnnotationBbox` can share a common sensibly named
   api for getting/setting the location of the text or box.
 
-    - `xyann` -> set the location of the annotation
-    - `xy` -> set where the arrow points to
-    - `anncoords` -> set the units of the annotation location
-    - `xycoords` -> set the units of the point location
-    - `set_position()` -> `Annotation` only set location of annotation
+    - *xyann* -> set the location of the annotation
+    - *xy* -> set where the arrow points to
+    - *anncoords* -> set the units of the annotation location
+    - *xycoords* -> set the units of the point location
+    - ``set_position()`` -> `.Annotation` only set location of annotation
```

Comment:
'.Annotation.set_position'? (quotes -> backticks)

Reply:
There is no `.Annotation.set_position`; it's on the underlying `Text`. But this is referring to the Annotation, and the fact that `.set_position` now works on it correctly, so I'm not sure we want to redirect to `Text`.